ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -696,6 +696,44 @@ def test_tensorflow_Atanh(  # NOQA
     )
 
 
+# AvgPool3d
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.AvgPool3D",
+    dtype_and_x=(ivy.Array, ivy.random_normal(shape=(1, 4, 4, 4, 3))),
+)
+def test_tensorflow_avg_pool_3d(
+    *,
+    dtype_and_x,
+    num_positional_args,
+    as_variable,
+    native_array,
+    frontend,
+    fn_tree,
+    on_device,
+):
+    input_dtype, x = dtype_and_x
+    ksize = [1, 2, 2, 2, 1]
+    strides = [1, 2, 2, 2, 1]
+    padding = 'VALID'
+    result = frontend.ops.tensorflow.raw_ops.AvgPool3D(
+        input=x,
+        ksize=ksize,
+        strides=strides,
+        padding=padding,
+        name=None,
+    )
+    helpers.test(
+        result=result,
```

Comment:
we should not calculate result and pass this here as this is not the proper way how we do testing, kindly see the docs
https://lets-unify.ai/docs/ivy/overview/deep_dive/ivy_frontends.html
https://lets-unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html